keyboards/3.0/ja-t-k0-kana.xml

```diff
@@ -18,53 +18,53 @@
         <key id="Jko" to="き"/>
 	</keys>
 
-	<layerMaps form="hardware">
-		<layerMap modifier="none">
+	<layers form="hardware" hardware="jis">
+		<layer modifier="none">
 			<row keys="1 2 3 4 5 6 7 8 9 0 minus caret yen" /> <!-- delete -->
 			<row keys="Jta Jtu Ji Jsu Jka Jn Jna Jni Jra Jse rough smooth" />
 			<row keys="Jsa Jto Jsi Jha Jki Jku Jma Jno Jri Jre Jke Jmu" />
 			<row keys="shift Jtu Jti Jso Jhi Jko Jre Jyo Jne Jru Jme Jro shift" />
 			<row keys="eigo space kata" />
-		</layerMap>
+		</layer>
 
-		<layerMap modifier="shift">
+		<layer modifier="shift">
 			<row keys="1 2 3 4 5 6 7 8 9 0 degree plus" />
 			<row keys="A Z E R T Y U I O P umlaut pound" />
 			<row keys="Q S D F G H J K L M percent micro" />
 			<row keys="greater-than W X C V B N question-mark period slash section" />
 			<row keys="space" />
-		</layerMap>
-	</layerMaps>
+		</layer>
+	</layers>
 
-	<layerMaps form="touch" minDeviceWidthMM="150">
+	<layers form="touch" minDeviceWidth="150">
 		<!-- optional attribute for min physical device size -->
-		<layerMap id="base">
+		<layer id="base">
 			<row keys="a z e r t y u i o p" />
 			<row keys="q s d f g h j k l m" />
 			<row keys="shift gap w x c v b n gap bksp" />
 			<row keys="numeric extra space enter" /> <!-- The default "space" definition includes `stretch="true"` -->
-		</layerMap>
+		</layer>
 
-		<layerMap id="shift">
+		<layer id="shift">
 			<row keys="A Z E R T Y U I O P" />
 			<row keys="Q S D F G H J K L M" />
 			<row keys="base W X C V B N bksp" />
```

Comment:
There is a gap after 'shift' and before 'bksp' on the base layer but not on the shift layer?